### DISCUSSION TITLE
cookie store expects conn.secret_key_base to be at least 64 bytes

### DISCUSSION THREAD

**Discussion by HenkVanMaanen** [plausible/analytics#3096] [Self-Hosted Support] [answered]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for self-hosted support

### Using official Plausible Cloud hosting or self-hosting?

Self-hosting

### Describe the bug

Version: v2.0.0-rc.3

Got error: cookie store expects conn.secret_key_base to be at least 64 bytes

Steps I took:
1. start plausible with env SECRET_KEY_BASE= less than 64 bytes, got that error
2. start plausible with more than 64 bytes, got that error
3. change plausible version to v2.0.0-rc.2, no error, plausible starts normally and can now register without problems

![image](https://github.com/plausible/analytics/assets/36051232/bf4fddb2-4162-4bd1-a701-06f60de9021c)

### Expected behavior

no error while loading register page

### Environment

```markdown
- OS:
- Browser:
- Browser Version:
```

**Comment by ruslandoga** [comment, marked as answer]
Hi @HenkVanMaanen 

Note that updating env vars and restarting a container doesn't change them in the container (they are read only at start time, immutability and all that). So you'd need to start a new container. And this is what happened when you used a different image.

**Reply by HenkVanMaanen** [reply]
A I see. Thanks for the clarification!
